corehq/ex-submodules/pillowtop/management/commands/run_ptop.py

```diff
@@ -36,6 +36,15 @@ class Command(NoArgsCommand):
                     dest='pillow_name',
                     default=None,
                     help="Run a single specific pillow name from settings.PILLOWTOPS list"),
+        make_option('--optimize',
```

Comment:
this should probably print out some message in the actual command to notify the user that they are running in optimized mode